_pages/security.md

```diff
@@ -47,13 +61,17 @@ class: bg-light-blue
     <div class="clearfix mxn1">
       <div class="col sm-col-12 my3">
         <img class="col sm-col-2 py1 pl3" src="{{ site.baseurl }}/assets/img/code.svg" alt="" width="113"/>
-        <p class="col sm-col-6 px3 regular py1 m0">We welcome external review of our privacy-protection measures. All of our code is available for public inspection in an open-source repository. Our goal: make sure that at every step users know their privacy is being protected by design.</p>
+        <p class="col sm-col-6 px3 regular py1 m0">
+          {% t security_page.footer.p_1 %}
+        </p>
         <div class="col sm-col-4 px2 py1 vt-dot-border">
           <div class="ml2">
             <div class='sm-show'>
               <img class="col col-6 mb1" src="{{ site.baseurl }}/assets/img/logo.svg" alt="login.gov">
             </div>
-            <p class="col col-12">For more information, please visit the login.gov <a href="{{site.baseurl}}/help">Help Center</a> or <a href="{{site.baseurl}}/contact">contact us</a>. You can also visit our <a target="_blank" href="https://github.com/18F/identity-idp">open-source repository</a>.</p>
+            <p class="col col-12">
```

Comment:
Maybe go back to the `markdown="1"` here. It looks like it isn't formatting it well, like you said.
<img width="344" alt="screen shot 2017-06-08 at 11 07 26 am" src="https://user-images.githubusercontent.com/4803473/26938760-c17081f0-4c3a-11e7-91bf-526242d8cc9b.png">